app/routes/events/utils.ts

```diff
@@ -272,21 +274,24 @@ export const eventStatusTypes = [
     label: 'Åpen (med påmelding)',
   },
 ];
+
 export const transformEventStatusType = (eventStatusType: string) => {
   return (
     find(eventStatusTypes, {
       value: eventStatusType,
     }) || eventStatusTypes[0]
   );
 };
+
 export const getEventSemesterFromStartTime = (
-  startTime: moment
+  startTime: Moment
```

Comment:
```suggestion
  startTime: Dateish
```
It is used with `event.startTime` which is `Dateish`